tikv/kv.go

```diff
@@ -582,6 +582,16 @@ type Scanner = txnsnapshot.Scanner
 // KVSnapshot implements the tidbkv.Snapshot interface.
 type KVSnapshot = txnsnapshot.KVSnapshot
 
+// MemDBRetriever implements unionstore.Retriever
+type MemDBRetriever = txnsnapshot.MemDBRetriever
+
+// EmptyRetriever implements unionstore.Retriever
+type EmptyRetriever = txnsnapshot.EmptyRetriever
```

Comment:
Why do we need `EmptyRetriever`? For the integration of the interfaces?

Reply:
`EmptyRetriever` is used to return an empty data set, i.e, the global temporary table.

Reply:
I didn't see the usage of `EmptyRetriever` in the other PR. Is it used?

Reply:
It is used in https://github.com/pingcap/tidb/pull/26948 . I think it is also ok to move it to tidb